src/platform/packages/shared/kbn-unified-data-table/src/components/data_table.tsx

```diff
@@ -1082,26 +1085,28 @@ const InternalUnifiedDataTable = React.forwardRef<
       }
 
       const leftControls = (
-        <>
+        <EuiFlexGroup>
```

Comment:
What impact do these changes have on the table header layout?

Reply:
Resolved in 1b592bb9af455d7fe101bddecae08e086431e8cf